### ISSUE TITLE
Create branding / style guide

### ISSUE THREAD

**Issue by Marie-L** [pyladies/project-communications#84]
**Summary**

**Acceptance Criteria**

**Do you want to work on this issue (Yes/No)?**

cc / @pyladies/communications

**Comment by Marie-L** [comment]
@cecivieira @julianafalves I will add the assets for the new repo here :

**Comment by cecivieira** [comment]
@Marie-L is not exactly a new repo. I will update this one https://github.com/pyladies/pyladies-assets

**Comment by Marie-L** [comment]
@cecivieira yeah sorry I referred to it as _'new'_ because there's nothing in it yet

**Comment by BethanyG** [comment]
@cecivieira - that repo belongs to the `tech & infra` team at the moment, and I think we want to leave it alone for now -- or submit PRs for work.  I think @Marie-L  is referring to this one:  https://github.com/pyladies/communications-assets, which **is** a new repo.

**Comment by Marie-L** [comment]
yes, thanks @BethanyG !

**Comment by BethanyG** [comment]
We should also look at how we are going to manage image assets.  Github has poor tools for diffing images.

**Comment by BethanyG** [comment]
_I spoke too soon!_  Turns out MS has added image diffing capability!  🎉  https://docs.github.com/en/github/managing-files-in-a-repository/rendering-and-diffing-images....just not for _design files._

**Comment by Marie-L** [comment]
@BethanyG it's a shame that it's not for design files, but I think we might have listed other options in another issue or notes. I'll have a look.

@cecivieira @julianafalves I've added the social media banners and avatar. I haven't included the event promos etc because each file has many layers so we would need to save them separately to make it easier for people to edit

**Comment by BethanyG** [comment]
@Marie-L  - version control for design files is either pegged to specific formats ([abstract](https://www.abstract.com/versions) & [kactus](https://github.com/kactus-io/kactus) for Sketch, [adobe experience manager](https://www.adobe.com/marketing/experience-manager.html) for Adobe, some other really expensive ones [pics.io](https://pics.io/pricing) & [sympli](https://sympli.io/versions) that support multiple file formats.  And then there is [Folio](http://folioformac.com/), which is limited to Mac users -- and it's $49 a pop.  There are others out there keyed to design pipelines and developer handoff -- each of course has its own subscription fees associated with it.  

Truth be told, design file formats are a can of worms because not everyone can afford design software, and the vendors have gone to great lengths to **not** cooperate with each other.

All that being said, we can probably provide several different formats for people to use -- and as long as we provide banners and text in SVG, they can be resized without losing clarity.

**Comment by cecivieira** [comment]
Perhaps, if we publish "branding colours, pyladies global avatars and pyladies global banners" in the Communication Assets Repository (https://github.com/pyladies/communications-assets) we'll duplicate the proposal of the general assets repository(https://github.com/pyladies/pyladies-assets), because one part of it description is "This repository is for all assets, logos, etc used for PyLadies.". 

So, maybe we could update the general repository with "branding colours, pyladies global avatars and pyladies global banners" and uses our Communication Assets Repository for to publish the editable graphics with instructions for translating the text beging the objective to facilitate the local communities its reuse.

What do you think?

**Comment by cecivieira** [comment]
@BethanyG we'll intend to publish temporarily the editable graphics, only during the context period. I didn't understand what kind of situation you tried to solved with the messages about tools for diffing images. Could you explain a little bit?
